content/blog/adoptopenjdk-virtual-roadshow/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: AdoptOpenJDK Virtual Roadshow
+date: "2020-07-08T12:00:00+00:00"
+author: georgeadams
+description: Join the AdoptOpenJDK virtual roadshow as we speak at a JUG near you!
+featuredImage: "./duke_mask.png"
+tags:
+  - AdoptOpenJDK
+  - Conferences
+---
+Join the AdoptOpenJDK virtual roadshow as we speak at a JUG near you!
+
+![Duke with mask](./duke_mask.png)
+
+AdoptOpenJDK is doing a virtual roadshow of the Java User Groups. We plan to reach as many groups as possible but we will also live stream the event on YouTube.
+
+## Abstract
+AdoptOpenJDK is rapidly becoming a leading provider of OpenJDK™ binaries. With over 170 million downloads in the last year, it is now a serious contender for your production usage of Java™. AdoptOpenJDK provides prebuilt OpenJDK binaries from a fully open source set of build scripts and infrastructure. This talk will cover how we build on over 15 different platforms, execute over 87 million tests and distribute OpenJDK binaries to millions of users. We will also cover how AdoptOpenJDK binaries compare against the Java binaries that you use today. If you're curious to understand the difference between OpenJDK, Oracle Java, AdoptOpenJDK and all the other distributions, then this is the talk for you!
```

Comment:
I think you can be a bit more bold in the opening sentence.  AdoptOpenJDK is *the* leading provider of OpenJDK binaries, and not only a _contender_ for production use, it is ready and in production in many enterprises!

Reply:
```suggestion
AdoptOpenJDK is the leading provider of OpenJDK™ binaries. With over 170 million downloads in the last year, it is successfully used by many enterprises and ready for your production usage of Java™. AdoptOpenJDK provides prebuilt OpenJDK binaries from a fully open source set of build scripts and infrastructure. This talk will cover how we build on over 15 different platforms, execute over 87 million tests and distribute OpenJDK binaries to millions of users. We will also cover how AdoptOpenJDK binaries compare against the Java binaries that you use today. If you're curious to understand the difference between OpenJDK, Oracle Java, AdoptOpenJDK and all the other distributions, then this is the talk for you!
```